src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -9,13 +9,16 @@
 5
 
 >> Value(ParseJSON("true"))
-Error({Kind:ErrorKind.InvalidArgument})
+1
 
 >> Value(ParseJSON("false"))
-Error({Kind:ErrorKind.InvalidArgument})
+0
 
 >> Value(ParseJSON("""5"""))
-Error({Kind:ErrorKind.InvalidArgument})
+5
+
+>> Value(ParseJSON("""123,456"""))
```

Comment:
In the discussions I remember seeing an optional second parameter to this overload, just like for the "normal" Value function, where we can specify the locale we want to use - for example, `Value(ParseJSON("""123,456"""), "fr-FR")` would return 123.456. @jorisdg can you comment on this?